source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -166,6 +166,14 @@ def output(self, key, obj):
     "address": AddressField()
 }
 
+geocode_poi = {
+    "embedded_type": Lit("poi"),
+    "quality": Lit("0"),
+    "id": fields.String(attribute='properties.geocoding.id'),
+    "name": fields.String(attribute='properties.geocoding.label'),
+    "poi": AddressField()
```

Comment:
I don't have adress nor poi_types from bragi.
AddressField generates exactly what we want.
I can create PoiField but it will be the same as AddressField.